src/rust/lib/data/src/text.rs

```diff
@@ -113,7 +113,7 @@ impl Display for Size {
 
 impl From<&str> for Size {
     fn from(text:&str) -> Self {
-        Size::new(text.len())
+        Size::new(text.chars().count())
     }
 }
```

Comment:
This is not a today change, but I have distaste for this conversion. What does it mean that string is _converted_ to its length? Those are completely different things.

Reply:
Agreed as discussed today. The impl has been removed in favor of `from_str` constructor.